badge-maker/lib/badge-renderers.js

```diff
@@ -10,6 +11,19 @@ function capitalize(s) {
   return `${s.charAt(0).toUpperCase()}${s.slice(1)}`
 }
 
+function colorsForBackground(color) {
+  if (brightness(color) <= 0.8) {
```

Comment:
It would be clearer what we're doing here if we gave this a name e.g:
```js
const brightnessThreshold = 0.8;
```
We should probably do the same thing for the logo thresholds too - those are also magic numbers.